modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -135,36 +135,39 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
-  name: 98-var-log-partition
+  name: 98-var-partition
 spec:
   config:
     ignition:
       version: 3.1.0
     storage:
       disks:
-      - device: /dev/nvme0n1
+      - device: /dev/<device_name> <1>
         wipeTable: false
         partitions:
-        - sizeMiB: 47000
-          startMiB: 47000
-          label: var-log
+        - sizeMiB: <partition_size>
+          startMiB: <partition_start_size> <2>
```

Comment:
`partition_start_offset` would be clearer.

Reply:
Resolved